.github/workflows/verify_samples.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Samples
+
+on: 
+  push:
+    branches: master
+  pull_request:
+    branches: '*'
+
+jobs:
+  linux:
+    name: Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y && sudo apt-get install -y flex bison clang lld
+      - name: Build
+        run: sh ./.ci_build_samples.sh
+  macOS:
+    name: MacOS
```

Comment:
Why is this MacOS instead of macOS?

Reply:
Stylized, is Apple's official way lower case even at the beginning on a sentence?

Reply:
I've pushed the fix for this now.